public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchMetricsQueryRunner.ts

```diff
@@ -99,13 +99,17 @@ export class CloudWatchMetricsQueryRunner extends CloudWatchRequest {
   interpolateMetricsQueryVariables(
     query: CloudWatchMetricsQuery,
     scopedVars: ScopedVars
-  ): Pick<CloudWatchMetricsQuery, 'alias' | 'metricName' | 'namespace' | 'period' | 'dimensions' | 'sqlExpression'> {
+  ): Pick<
+    CloudWatchMetricsQuery,
+    'alias' | 'metricName' | 'namespace' | 'period' | 'dimensions' | 'sqlExpression' | 'expression'
+  > {
     return {
       alias: this.replaceVariableAndDisplayWarningIfMulti(query.alias, scopedVars),
       metricName: this.replaceVariableAndDisplayWarningIfMulti(query.metricName, scopedVars),
       namespace: this.replaceVariableAndDisplayWarningIfMulti(query.namespace, scopedVars),
       period: this.replaceVariableAndDisplayWarningIfMulti(query.period, scopedVars),
-      sqlExpression: this.replaceVariableAndDisplayWarningIfMulti(query.sqlExpression, scopedVars),
+      expression: this.templateSrv.replace(query.expression, scopedVars),
+      sqlExpression: this.templateSrv.replace(query.sqlExpression, scopedVars, 'raw'),
```

Comment:
Not sure about removing multi warning for sql?

Reply:
It should be fine? It's a query string field if I'm not wrong, so I think there are legitimate reasons to use a multi variable

Reply:
Is that necessary for the bug fix, I thought we only needed to add expression? Also what does adding "raw" as a third argument do?

Reply:
It's not necessary, I wanted to make it more similar to [this one](https://github.com/grafana/grafana/blob/da4c12b0d6021f3669dd482f9be0ef0584241918/public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchMetricsQueryRunner.ts#L181), but I guess it can be done as part of [this](https://github.com/grafana/oss-plugin-partnerships/issues/149)